README.md

```diff
@@ -123,152 +125,244 @@ Congratulations! You can now send remote push notifications to your device from
-			[IterableAPI initializeWithApiKey:@"YOUR API KEY" launchOptions:launchOptions config:config];
-			...
-		}
-		
-		- (BOOL)handleIterableURL:(NSURL *)url context:(IterableActionContext *)context {
-			// Assuming you have a DeeplinkHandler class that handles all deep link URLs and navigates to the right place in the app
-			return [DeeplinkHandler handleUrl:url];
-		}
-		```
-		
-	* Handling Email Links
-		
-		For Universal Links to work with link rewriting in emails, you need to set up apple-app-site-association file in the Iterable project. More instructions here: [Setting up iOS Universal Links](https://support.iterable.com/hc/en-us/articles/115000440206-Setting-up-iOS-Universal-Links).
+	[IterableAPI getAndTrackDeeplink:iterableLink callbackBlock:^(NSString* originalURL) {
+		//Handle Original URL deeplink here
+	}];
 
-		When an email link is clicked your `UIApplicationDelegate`'s [application:continueUserActivity:restorationHandler:](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623072-application?language=swift) method is called. If you already have an Iterable `urlDelegate` defined (see *Handling Links from Push Notifications* section above), the same handler can be used for email deep links by calling `handleUniversalLink:`.
+	return true;
+}
+```
+	
+##### 3. In-App Notifications
 
-		Swift:
-		
-		```swift
-		func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([Any]?) -> Void) -> Bool {
-			guard let url = userActivity.webpageURL else {
-				return false
-			}
-
-			// This will track the click, retrieve the original URL and call `handleIterableURL:context:` with the original URL
-			return IterableAPI.handle(universalLink: url)
-		}
-		```
-
-		Objective-C:
-		
-		```objective-c
-		- (BOOL)application:(UIApplication *)application continueUserActivity(NSUserActivity *)userActivity restorationHandler:(void (^)(NSArray *restorableObjects))restorationHandler {
-			// This will track the click, retrieve the original URL and call `handleIterableURL:context:` with the original URL
-			return [IterableAPI handleUniversalLink:userActivity.webpageURL];
-		}
-		```
-		
-		Alternatively, call `getAndTrackDeeplink` along with a callback to handle the original deeplink url. You can use this method for any incoming URLs, as it will execute the callback without changing the URL for non-Iterable URLs.
+###### Default behavior
+By default, when an in-app message arrives from the server, the SDK automatically shows it if the app is in the foreground. If an in-app message is already showing when the new message arrives, the new in-app message will be shown 30 seconds after the currently displayed in-app message closes (see how to change this default value below). Once an in-app message is shown, it will be "consumed" from the server queue and removed from the local queue as well. There is no need to write any code to get this default behavior. 
```

Comment:
- maybe use an anchor link to jump to the below section about how to change the value of the delay